### PR TITLE
Add build instructions

### PR DESCRIPTION
**Pull request by dgwatkins**
Address #8

I also took the chance to fix the checks (github actions)

### CODE REVIEW

**Comment by dgwatkins** [comment]
Please review @OnTheGoSystems/compatibility

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -4,10 +4,15 @@ on: [push, pull_request]
...
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.0
     steps:
       - uses: actions/checkout@v2
 
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+
```
Comment (strategio) [inline .github/workflows/php.yml:15]: I guess you referred to that https://github.com/marketplace/actions/setup-php-action for this change, right @dgwatkins?

**Review by strategio — DISMISSED** [review]
Looks good @dgwatkins +1

File: .github/workflows/php.yml (hunk 1 above)
Comment (dgwatkins) [reply .github/workflows/php.yml:15]: Yes, thats right @strategio 

But the action didn't run, its been queued for 15h

I will have to look for a different approach (update PHPUnit maybe?)

Comment (dgwatkins) [reply .github/workflows/php.yml:15]: I was using the wrong Ubuntu image.
Now it passes.

I made an extra commit so I will need your re-approval @strategio

**Review by strategio — APPROVED** [review]
+1 @dgwatkins

**Pull request merged**